frontend/pages/ManageControlsPage/OSSettings/OSSettings.tsx

```diff
@@ -40,17 +39,21 @@ const OSSettings = ({
     data: aggregateProfileStatusData,
     refetch: refetchAggregateProfileStatus,
     isLoading: isLoadingAggregateProfileStatus,
-  } = useQuery<ProfileSummaryResponse>(
+  } = useQuery(
     ["aggregateProfileStatuses", teamId],
-    () => mdmAPI.getAggregateProfileStatuses(teamId),
+    () =>
+      mdmAPI.getAggregateProfileStatuses(teamId, config?.mdm_enabled ?? false),
     {
       refetchOnWindowFocus: false,
       retry: false,
     }
   );
 
   // MDM is not on so show messaging for user to enable it.
-  if (!config?.mdm.enabled_and_configured) {
+  if (
+    !config?.mdm.enabled_and_configured &&
+    !config?.mdm.windows_enabled_and_configured
```

Comment:
Does this need to be updated?

Reply:
will be updated afterwards I think, for now we just want to make the page accessible for windows.

Reply:
(in order to turn on disk encryption)